extensions/panache/rest-data-panache/runtime/src/main/resources/META-INF/quarkus-extension.yaml

```diff
@@ -0,0 +1,12 @@
+---
```

Comment:
Why do we need this file? This module cannot be used on its own and only contains the common classes that are used by the Hibernate ORM REST Data extension (and the future Mongo alternative).

Reply:
First of all, it's not exposed as an extension  to the users. It is in fact not required in this case. I just added it for consistency since the project is following the extension project structure.